systemtest/src/main/java/io/strimzi/systemtest/Environment.java

```diff
@@ -54,6 +54,10 @@
      * CO reconciliation interval.
      */
     private static final String STRIMZI_FULL_RECONCILIATION_INTERVAL_MS_ENV = "STRIMZI_FULL_RECONCILIATION_INTERVAL_MS";
+    /**
+     * Openshift 4 parameter.
+     */
+    private static final String OPENSHIFT4_ENV = "OPENSHIFT4";
```

Comment:
Why does the correct functioning of the strimzi upgrade depend on the OpenShift4-ness of the cluster?

Reply:
Upgrade from `0.8.2` to `0.11.4` is not working on ocp4 and I didn't find better solution how to determine which cluster version we are using in the tests.

Reply:
Is there an issue about that? I don't see one. We should look into understanding why it doesn't work on that platform, and only add a _temporary_ workaround like this if and when we know that it's not a simple fix.

Reply:
Realistically, 0.8.2 existed long before OpenShift 4. And I do not think we will do any reelases for such an old version to get the upgrades working. I guess the only interesting question is whether it is OpenShift 4+ or Kubernetes 1.12+.